### DISCUSSION TITLE
Can not able get element id value ? Any possible way to get ?

### DISCUSSION THREAD

**Discussion by Lakshmanan4233** [appium/appium#18934] [Q&A]
![Snag_b442d9](https://github.com/appium/appium/assets/96466534/4ab8c31d-5539-4b84-b70b-54de8cd05245)

**Comment by jlipps** [comment]
That is the ID that appium generates for the element. It's not useful outside of an Appium client. It looks like the element you have selected does not have any kind of unique id attached to it, so a developer would need to add a resource-id or content-desc.
